examples/cross-contract-low-level/tests/general.rs

```diff
@@ -0,0 +1,74 @@
+use near_sdk_sim::{
+    call, deploy, init_simulator, to_yocto, ContractAccount, UserAccount, DEFAULT_GAS,
+    STORAGE_AMOUNT,
+};
+extern crate cross_contract_low_level;
+use cross_contract_low_level::CrossContractContract;
+
+near_sdk_sim::lazy_static::lazy_static! {
+    static ref TOKEN_WASM_BYTES: &'static [u8] = include_bytes!("../res/cross_contract_low_level.wasm").as_ref();
+}
+
+fn init(
+    initial_balance: u128,
+) -> (UserAccount, ContractAccount<CrossContractContract>, UserAccount) {
+    let master_account = init_simulator(None);
+    let contract_account = deploy! {
+    contract: CrossContractContract,
+    contract_id: "contract",
+    bytes: &TOKEN_WASM_BYTES,
+    signer_account: master_account
+    };
```

Comment:
This is my least favorite version of how to pass arguments to `deploy!` shown here, given the improper indentation on lines 17-20.

Reply:
I'm confused, do you mean that you just don't like the indentation?

Reply:
It seems/seemed like you're demonstrating three ways to call the `deploy!` macro:

1. with positional arguments, `deploy!(master_account, "contract", ...)`
2. with named arguments, using parentheses: `deploy!(signer_account: master_account, contract_id: "contract", ...)`
3. with named arguments, using curly brackets and no parentheses: `deploy! { signer_account: master_account, contract_id: "contract", ... }`

I'm saying that I don't think there's value in supporting more than one, and I think the parentheses seem like a fine way to go. Judging by the improper indentation here, it seems like rustfmt doesn't automatically set indentation using this curly-bracket version.